src/test/java/org/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -133,8 +133,10 @@ void findMissingLocalizationKeys() throws IOException {
                            .collect(Collectors.joining("\n",
                                    """
 
-                                           DETECTED LANGUAGE KEYS WHICH ARE NOT IN THE ENGLISH LANGUAGE FILE
-                                           PASTE THESE INTO THE ENGLISH LANGUAGE FILE "JabRef_en.properties"
+                                           DETECTED LANGUAGE KEYS WHICH ARE NOT IN THE ENGLISH LANGUAGE FILE.
+                                           PASTE THESE INTO THE ENGLISH LANGUAGE FILE "JabRef_en.properties".
+                                           Search for a proper place; typically related keys are grouped together.
+                                           Maybe you find a similar key - then adapt your langauge instead of adding load to translators by adding a new key.
```

Comment:
```suggestion
                                           If a similar key is already present, please adapt your language instead of adding load to translators by adding a new key.
```

Reply:
Heavily depends on the context, something is maybe a menu context

Reply:
We do not support any distinction for menus any more. - This needs to be re-added. See https://github.com/JabRef/jabref-issue-melting-pot/issues/657

Thus, we can just go ahead with this one. - Currently, the contributors invent new terms, which is not good.